### DISCUSSION TITLE
Comments using Disqus

### DISCUSSION THREAD

**Discussion by rohitfarmer** [gethinode/hinode#454] [Help] [answered]
Is there a way to enable commenting via Disqus instead of Utterance?

**Comment by myrthos** [comment, marked as answer]
Hugo has out of the box support for Discus, see [here](https://gohugo.io/content-management/comments/). It looks very easy to add. The  template call should be added to `layouts/partials/assets/comments.html` instead of the utterances code in that file.  
Besides those changes the Content-Security-Policy in `server.html` would need to be changed to enable loading the script  and styles. Pressing F12 in a Chromium based browser shows what would be required, including any hashes that might need to be added.

That said, there is apparently an [issue](https://gohugo.io/templates/internal/#disqus) when running Hugo server, which might need an additional change and it is not quite clear how to make sure that discuss follows the selected light/dark style.

Might be a nice Hinode module in the future :grin:

**Reply by 0xtiago** [reply]
Hi @myrthos !
Do you have, or, have you seen any example of Disqus integration in any site with Hinode template. I couldn't figure out how to solve this issue... :(

**Comment by dims31** [comment]
Hello,
Adding Disqus could be a very nice feature for blogging purpose (github authentication is not really appreciated by my friends  ^^)

I tried to add disqus with partial without success...

If someone have an idea : https://deploy-preview-41--tourdesminous.netlify.app/destinations/indonesie/2023-10-22-ubud-munduk/

Github : https://github.com/dims31/tdm/tree/feat-disqus

Any help will be really appreciated !

**Reply by dims31** [reply]
Fixed : Add Disqus related URL in Content-Security-Policy in server.toml and netlify.toml
